.github/workflows/deploy-stage.yaml

```
@@ -29,10 +29,10 @@ jobs:
         with:
           aws-region: ${{ secrets.AWS_REGION }}
           role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/github
-      - name: List Parameter Store services
+      - name: Run terraforming THE on stage cluster
         env:
           USE_AWS_VAULT: false
         run: |
-          source ./scripts/lib/external_config.sh
-          init_chamber
-          run_chamber list-services
+          set -euo pipefail
```

nit: for the below commands this should be enough.
```suggestion
          set -e
```

---

Yeah, I'm being "future-proof" :sweat_smile:

---

I mean, scripts tend to grow organically and people (myself included) often forget to extend the range of useful checks.